dashboard/test/models/script_level_test.rb

```diff
@@ -277,6 +277,7 @@ class ScriptLevelTest < ActiveSupport::TestCase
     student = create :student
     I18n.locale = 'non-default-locale'
     assert_equal script_levels[1].path, script_levels[0].next_level_or_redirect_path_for_user(student)
+    I18n.locale = I18n.default_locale
```

Comment:
Is there a way to do this in an after_each sort of method in ruby? Right now, if the assert on line 279 were to fail, the locale would not get reset.